ReactiveCocoa/Swift/Signal.swift

```diff
@@ -125,7 +125,8 @@ public final class Signal<Value, Error: ErrorType> {
 	///
 	/// Returns a Disposable which can be used to disconnect the observer. Disposing
 	/// of the Disposable will have no effect on the Signal itself.
-	public func observe(observer: Observer) -> Disposable? {
+	@discardableResult
```

Comment:
Ditto: this shouldn't be discardable. Users should ignore it using `_ = ` if they really want to.